src/deps_deploy/deps_deploy.clj

```diff
@@ -88,22 +93,81 @@
                  :coordinates (mvn-coordinates coordinates))
   (println "done."))
 
-(defmethod deploy :local [{:keys [artifact-map coordinates]}]
+(defmethod deploy* :local [{:keys [artifact-map coordinates]}]
   (println "Installing" (artifact coordinates) "to your local `.m2`")
   (aether/install :artifact-map artifact-map
                   :transfer-listener :stdout
                   :coordinates (mvn-coordinates coordinates))
   (println "done."))
 
-(defn -main [deploy-or-install artifact & [sign-releases]]
-  (let [pom (slurp "pom.xml")
-        coordinates (coordinates-from-pom pom)
-        versioned-pom (spit (versioned-pom-filename coordinates) pom)]
-
+(defn- tag-val-replacer [tag->val]
+  (fn [{:keys [tag content attrs] :as el}]
+    (let [replacement-val (tag->val tag ::default)]
+      (if (= ::default replacement-val)
```

Comment:
This is a great use-case for `cond->`:
```Clojure
(cond-> el
   (not= ::default replacement-val) (assoc :content replacement-val))
```

It's almost like this is an `update` right?

```clojure
(update el :content (fn [c] (if (= ::default replacement-val) c replacement-val)))
```

Reply:
Indeed it is.  Will revise both of those.  Thanks for caring.